src/main/java/com/khoros/twitterapp/services/TwitterService.java

```diff
@@ -24,8 +22,9 @@
     public static final String GENERAL_ERR_MSG = "Whoops! Something went wrong. Try again later.";
     public static final String NO_TWEET_TEXT_MSG = "No tweet text entered.";
     public static final String TWEET_TOO_LONG_MSG = "Tweet text surpassed " + TwitterService.MAX_TWEET_LENGTH + " characters.";
-    private CacheUp cacheUp;
     public Twitter twitterFactory;
+    public enum CacheListType { HOME, USER };
```

Comment:
if you are going with an enum, this should be pulled out since the cache uses the datatype as well.  With the way your code is, the cache is directly tied to TwitterService.CacheListType which is odd